src/app.md

```diff
@@ -119,7 +119,22 @@ Your favorite number must be 256.
 
 [![log-badge]][log] [![env_logger-badge]][env_logger] [![cat-debugging-badge]][cat-debugging]
 
-[Write me!](https://github.com/brson/rust-cookbook/issues/61)
+```rust
+#[macro_use] extern crate log;
+extern crate env_logger;
+
+fn main() {
+    env_logger::init().unwrap();
```

Comment:
Ok then. I guess other logger examples should be updated for consistency sake as well as a note in https://brson.github.io/rust-cookbook/about.html#a-note-about-error-handling might be needed. Something along the lines: `unless explicitly suggested by the crate docs`;)